tests/integ/scala/test_function_suite.py

```diff
@@ -2928,6 +2928,10 @@ def test_array_remove(session):
     )
 
 
+@pytest.mark.skipif(
+    "config.getoption('local_testing_mode', default=False)",
+    reason="array_remove is not yet supported in local testing mode.",
+)
```

Comment:
you added this to the wrong test. You want to move this before `test_to_array`.

also cc: @sfc-gh-jrose / @sfc-gh-aling

Reply:
`array_remove` is not yet supported by local testing, but I don't see anything in these tests that would require this mark.

Reply:
the new implementation would indeed break local testing in that it uses `array_construct` not `to_array` -- `array_construct` hasn't been patched.

@sfc-gh-lninobrijaldo could you help implement `array_construct` in snowpark.mock._functions?
you can take reference of how `to_array` is implemented [here](https://github.com/snowflakedb/snowpark-python/blob/main/src/snowflake/snowpark/mock/_functions.py#L1396).

if you have question about local testing, you can start from this page: https://docs.snowflake.com/en/developer-guide/snowpark/python/testing-locally and reach out to me or Jamison for help